### PR TITLE
Port mappings to tidy eval

### PR DESCRIPTION
**Pull request by lionel-**

### CODE REVIEW

**Comment by lionel-** [comment]
@thomasp85 is there any ggplot extension out there that makes assumptions about the structure of `params$facets`, i.e. that it's a `quoted` object from the plyr package?

**Comment by thomasp85** [comment]
Maybe...

Good news is that if there are, they are all in one of my packages with a very high probability - I'll make sure to take care of these...

File: DESCRIPTION [hunk 1]
```diff
@@ -52,7 +52,7 @@ Suggests:
 Remotes: hadley/scales,
     hadley/svglite,
     jimhester/withr,
-    tidyverse/rlang
+    lionel-/rlang@1def2e2cffaaa
```
Comment (hadley) [inline DESCRIPTION]: I assume there will be an rlang PR soon?

Comment (lionel-) [reply DESCRIPTION]: There is already one: https://github.com/r-lib/rlang/pull/471

**Comment by lionel-** [comment]
> I didn't read the code too carefully as I assume it works.

I couldn't run the doppelganger tests because I don't have the right deps on my machine though :/. The default legends are not deparsed correctly (you see the `~` of quosures). I'm going to fix this before merging, but it'd be nice to run the visual tests properly before release. Maybe it's time to finally fix vdiffr.

**Pull request merged**